docs/index.rst

```diff
@@ -2,10 +2,32 @@
 IRISPy Documentation
 ********************
 
-This is the documentation for IRISPy.
+.. warning::
+    
+    IRISPy is still under heavy development and has not yet seen its first
+    release. If you want to help test IRISPy follow these `installation
+    instructions
+    <https://github.com/sunpy/irispy/wiki/IRISpy-Installation-Instructions>`__.
 
-Reference/API
-=============
+IRISPy is a SunPy-affiliated package that provides the tools to read in and
+analyze data from the `IRIS <http://iris.lmsal.com>`__ solar-observing
+satellite in Python.
 
-.. automodapi:: irispy
-.. automodapi:: irispy.spectrograph
+The Interface Region Imaging Spectrograph is a NASA-funded Small Explorer which
+uses a high-frame-rate ultraviolet imaging spectrometer to make observations of
+the Sun. It provides 0.3 arcsec angular resolution and sub-angstrom spectral
+resolution at a one second cadence.
```

Comment:
Remove comment about one second cadence.  The cadence depends on the observing mode.  We don't have to go into the details of that here.